plugins/CoreHome/angularjs/common/directives/autocomplete-matched.js

```diff
@@ -19,26 +19,30 @@
     angular.module('piwikApp.directive').directive('piwikAutocompleteMatched', piwikAutocompleteMatched);
 
     function piwikAutocompleteMatched() {
-        return function(scope, element, attrs) {
-            var searchTerm;
 
-            scope.$watch(attrs.piwikAutocompleteMatched, function(value) {
-                searchTerm = value;
-                updateText();
-            });
+        return {
+            priority: 10,
```

Comment:
Basically this is the fix. We tell the directive to render after other directives